bigbluebutton-html5/public/resources/images/video-menu/icon-swap.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think these 2 icons should be imported from bigbluebutton-font (or added there first if they are not there yet)
@tylercopeland what do you think?
[Icon 1](https://github.com/Tainan404/bigbluebutton/blob/be04f6c701fc982a3f527d3e32391dd40601593b/bigbluebutton-html5/public/resources/images/video-menu/icon-swap.svg) [Icon 2](https://github.com/Tainan404/bigbluebutton/blob/be04f6c701fc982a3f527d3e32391dd40601593b/bigbluebutton-html5/public/resources/images/video-menu/icon-webcam-off.svg)

Reply:
I dont think thats possible unless we use 2 different icons, since this image has 3 colors.

Reply:
@OZhurbenko I agree with @oswaldoacauan. I'm not sure if the the icon-font can / should support a complex illustrations like this. I'd have to test it out. 

@GhaziTriki what are your thoughts on this? Is this something we should look into doing? 

@OZhurbenko @oswaldoacauan for context, @GhaziTriki has a ton of experience building fonts and he supported me with the creation of the bbb icon font.

Reply:
If the middle part was the same colour as the background you would be able to get away with a font icon, but you definitely can't do three colours with a font. A complex graphic is better as a custom SVG I think.